README.Rmd

```diff
@@ -48,6 +48,40 @@ incubation_H7N9 <- epiparameter::epidist(
   delay_dist = "incubation"
 )
 
-# Plot probability mass function
-plot(0:10,incubation_H7N9$pmf(0:10),xlab="time since infection")
+# Plot probability distributions
+plot(incubation_H7N9)
 ```
+
+## `epiparameter` tutorial with applications
+
+For a full explanation of the functions and data stored in the `epiparameter` package and a few possible applications of the delay distribution data please see the [introduction to `epiparameter`](https://epiverse-trace.github.io/epiparameter/articles/epiparameter.html).
+
+## Contributing to library of delay distributions
+
+If you would like to contribute to the different delay distributions stored in the `epiparameter` package, you can access the [google sheet](https://docs.google.com/spreadsheets/d/1VzC8NFHSQYPKsiDnxA7gDsm4cVr79outKQZJP8cC7Ro/edit?usp=sharing) and add your data.
+
+Notes on the the spreadsheet:
+
+- Most studies will report the distribution(s) they fitted to the data. In the case that the study did not report the distribution of the reported parameters or summary statistics (mean and standard deviation of the distribution) the distribution can be assumed (e.g. by looking at the shape of the distribution from figures in the paper). When a distribution is being assumed and not explicitly stated please add this assumption to the notes section of the spreadsheet.  We are only interested in studies that report the distribution fitted to the data. This will ensure that if this data is used in another application the user can be alerted that the distribution is assumed and not known with certainty.
```

Comment:
Perhaps rephrase to "We are currently only accepting studies that report the distribution ..."

Reply:
The vignette said something similar to what you suggested in a previous version but we decided to updated to give more freedom to assumed distributions. https://github.com/epiverse-trace/epiparameter/blob/d104b0fdb42fd53cc360acfcadd2f3ebeef872bd/vignettes/epiparameter.Rmd#L455